Sources/FuzzilliCli/main.swift

```diff
@@ -290,8 +339,19 @@ func makeFuzzer(for profile: Profile, with configuration: Configuration) -> Fuzz
                                   inliningPolicy: InlineOnlyLiterals(),
                                   ecmaVersion: profile.ecmaVersion)
 
+    // The evaluator to score produced samples.
+    let evaluator = ProgramCoverageEvaluator(runner: runner, maxResetCount: UInt64(maxResetCount))
+
     // Corpus managing interesting programs that have been found during fuzzing.
-    let corpus = Corpus(minSize: minCorpusSize, maxSize: maxCorpusSize, minMutationsPerSample: minMutationsPerSample)
+    let corpus: Corpus
+    switch corpusName {
+    case "basic":
+        corpus = BasicCorpus(minSize: minCorpusSize, maxSize: maxCorpusSize, minMutationsPerSample: minMutationsPerSample)
```

Comment:
Nit: can you avoid the code duplication here? I think you can just `fatalError` in the default case since that should never happen